syntax = "proto3";
import "google/protobuf/empty.proto";
import "common.proto";

service ImageProcessor {
  rpc ReadFileSize(FileSize) returns (google.protobuf.Empty);
  rpc ProcessImage(stream FileChunk) returns (stream ProgressResponse);
}
